EventsExpress/ClientApp/src/components/event/event-filter-status.js

```diff
@@ -16,8 +16,11 @@ function eventStatusHistoryReadingString(option) {
 class EventFilterStatus extends Component {
     checkboxGroup() {
         let { options, input } = this.props;
+        if (input.value != "")
+            input.value;
```

Comment:
What does this line mean? You simply call input.value, but it has no effect. How about removing it?